app/javascript/components/supplemental_application/sections/LeaseInformationInputs.js

```diff
@@ -52,7 +58,8 @@ const LeaseInformationInputs = ({ formApi, store }) => {
         />
         <CustomFormGrid.Select
           label='Preference Used'
-          field='preference_used' options={[]} placeholder='Select One' />
+          onChange={toggleNoPreferenceUsed(formApi)}
```

Comment:
@fddayan I wrote this code so responding -- So you're saying it should be `onChange={toggleNoPreferenceUsed}` instead of `onChange={toggleNoPreferenceUsed(formApi)}`, I remember I tried using the former (`{toggleNoPreferenceUsed}`) but it wasn't working 100% correctly.